instance/management/commands/instance_statistics_csv.py

```diff
@@ -131,7 +137,10 @@ def handle(self, *args, **options):
             ))
             sys.exit(1)
 
-        self.collect_instance_statistics(out, domains, start_date, end_date)
+        if options['users']:
+            self.collect_active_users(out, domains, start_date, end_date)
```

Comment:
@janimo, as stated [here](https://github.com/open-craft/opencraft/pull/768/files#r611534309), this should be an addition to the existing functionality of the command and not something separate.